### PR TITLE
[DOCU-1924] New "Manage decK with Docker" page

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Summary

Created a new page that explains how to use the Docker image to manage decK.

Questions for reviewers:
- Do the titles/headers make sense?
- Did I do all the necessary steps when adding a new page (I think it just had to be added to the nav page)?
- Where in "Guides" should this live?

### Reason

[DOCU-1924](https://konghq.atlassian.net/browse/DOCU-1924)
### Testing

https://deploy-preview-4124--kongdocs.netlify.app/deck/latest/guides/manage-with-docker/

### CODE REVIEW

File: src/deck/guides/manage-with-docker.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+content_type: how-to
+---
+
+If you used the `kong/deck` Docker image to install decK, you can use the Docker image to manage decK. 
+
+## Prerequisites
+decK must be installed using a [Docker image](/deck/latest/installation/#docker-image).
+
+## Export the configuration
+Run a command similar to the following to export the `kong.yaml` file:
```
Comment (Guaris) [inline src/deck/guides/manage-with-docker.md:12]:
```suggestion
Run this command to export the `kong.yaml` file:
```

File: src/deck/guides/manage-with-docker.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+```bash
+docker run -i \
+-v $(pwd):/deck \
+--workdir /deck \
+kong/deck --kong-addr http://example:KONG_ADMIN_PORT:8001 --headers kong-admin-token:KONG_ADMIN_TOKEN dump --all-workspaces
+```
+
+## Reset the configuration
+Run a command similar to the following to initial Kong objects:
```
[the remark this answers is not in the document]
Comment (Guaris) [reply src/deck/guides/manage-with-docker.md:32]: Unless initial here means something different

File: src/deck/guides/manage-with-docker.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+```bash
+docker run -i \
+-v $(pwd):/deck \
+kong/deck --kong-addr http://example:KONG_ADMIN_PORT:8001 --headers kong-admin-token:KONG_ADMIN_TOKEN reset
+```
+
+## Import the configuration
+Run a command similar to the following to import `kong.yaml`:
```
Comment (Guaris) [inline src/deck/guides/manage-with-docker.md:41]:
```suggestion
Run this command to import `kong.yaml`:
```

**Review by Guaris — DISMISSED** [review]

File: app/_data/docs_nav_deck_1.12.x.yml [hunk 4]
```diff
@@ -48,6 +48,8 @@ items:
         url: /guides/defaults
       - text: Using environment variables with decK
         url: /guides/environment-variables
+      - text: Managing decK with Docker
```
Comment (mheap) [inline app/_data/docs_nav_deck_1.12.x.yml:51]: Could we rename to `Run decK with Docker`? decK is the tool you use to manage Gateway configuration. You're not managing deck itself

**Review by lena-larionova — COMMENTED** [review]
Can you please backport this as far as the single-sourcing goes? The kong/deck image has been available since long before we started writing docs about decK: https://hub.docker.com/r/kong/deck/tags

File: app/_data/docs_nav_deck_pre-1.7.yml [hunk 5]
```diff
@@ -37,6 +37,8 @@ items:
         url: /guides/best-practices
       - text: Using decK with Kong Gateway (Enterprise)
         url: /guides/kong-enterprise
+      - text: Run decK with Docker
+        url: /guides/run-with-docker
```
Comment (lena-larionova) [inline app/_data/docs_nav_deck_pre-1.7.yml:41]:
```suggestion
```
Take it out of this one - there's something up with the pre-1.7 doc set that we need to fix + it's not single-sourced.

**Review by lena-larionova — APPROVED** [review]
One minor edit to remove the topic from the `pre-1.17` nav, otherwise LGTM

**Pull request merged**